src/style.css

```diff
@@ -0,0 +1,70 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #eee;
+}
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  
+  height: 60px;
+  margin-top: 20px;
+  margin-left: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  position: relative;
+}
+
+.link__logo {
+  position: absolute;
+  left: 50px;
+}
+
+.nav {
+  height: 100%;
+}
+
+.nav__list {
+  height: 100%;
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0 50px 0 0;
+}
+
+.nav__item {
+  margin: 0 10px;
+  position: relative;
+}
+
+.nav__link {
+  height: 100%;
+  color: black;
+  display: flex;
+  align-items: center;
+}
+
+.nav__link:link {
+  text-decoration: none;
+  text-transform: uppercase ;
+}
+
+.nav__link:hover::after,
```

Comment:
on design, you should change the color of letters on hover not adding the line below